lib/usermanager.js

```diff
@@ -53,6 +52,20 @@ exports = module.exports = {
     'DuplicateUserError': DuplicateUserError
   },
 
+  /**
+   * checks if a user is permitted the action on the resource
+   *
+   * @param {string} action
+   * @param {object} user
+   * @param {callback} next (function (err, isAllowed))
+   */
+
+  hasPermission: function(action, user, next) {
+    permissions = require('./permissions');
+    var resource = permissions.buildResourceString(user.tenant._id, '/api/user/resetpassword');
```

Comment:
Now that I look at this shouldn't this resource be injected into hasPermission? 

Something like 
```
hasPermission: function(action, user, resource, next) {
  permissions = require('./permissions');
  var resource = permissions.buildResourceString(user.tenant._id, resource);
// etc.
```
Then on line 767
```
self.hasPermission('update', user, '/api/user/resetpassword', function(err, isAllowed) {
   // do stuff
}
```